client/src/components/ResultPanel/ResultPanel.ts

```diff
@@ -42,4 +69,5 @@ export const showResult = (html: string, uri?: Uri, title?: string) => {
     );
   }
   resultPanel.webview.html = html;
+  resultPanels[vscodeContext.uuid] = resultPanel;
```

Comment:
Here we're re-using previous resultPanel. So there's already a uuid entry referring the instance.
When dispose, it only deletes the initial uuid entry. So it looks like preserving disposed references. Not sure if it's a memory leak issue.

Reply:
Not sure if I understand what you're saying. Can you elaborate?

Reply:
When the first time show result, say `vscodeContext.uuid === 001`. We create a resultPanel instance say `instance1`.
So
```
resultPanels === {
    '001': instance1
}
```
And we register
```
instance1.onDidDispose(() => {
    delete resultPanels['001'];
});
```
When the 2nd time show result, say `vscodeContext.uuid === 002`. And user set `singlePanel === true`. The `instance1` is re-used.
So
```
resultPanels === {
    '001': instance1
    '002': instance1
}
```
Then when instance1 disposed, the `resultPanels` still has entries referring to `instance1`.
```
resultPanels === {
    '002': instance1
}
```

Reply:
Ah, got it. Thanks for explaining. I've updated things to use `viewType` (the first parameter in `createWebviewPanel`) to store result panels and have verified things are stored as expected in https://github.com/sassoftware/vscode-sas-extension/pull/550/commits/0e50c8d79562e2a63a66d2d8d8f62042df51cc13